### PR TITLE
Adds Notification Stream Kind Icons

### LINKED ISSUE #4241: Notifications Control Center: Stream Icons
Let's add a left icon to each one of the available streams, as follows:

Thanks @drw158 !

screen shot 2015-08-17 at 11 58 28 am

### PR DESCRIPTION
**Pull request by jleandroperez**
#### Steps:
1. Log into your WordPress.com account
2. Tap over `Me` > `Notification Settings`
3. Tap over any of your blogs (OR the row named `Comments on Other Sites`)
4. A screen with the available notification streams should be visible

Each one of the rows, now, should also display an icon on the left side of the cell, as shown in the screenshot attached below.

Needs Review: @astralbodies (Thanks in advance!)
Closes #4241

sample

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingStreamsViewController.swift [hunk 1]
```diff
@@ -155,6 +156,20 @@ public class NotificationSettingStreamsViewController : UITableViewController
...
+    private func imageForStreamKind(streamKind: NotificationSettings.Stream.Kind) -> UIImage? {
+        let imageName : String
+        switch streamKind {
+        case .Email:
+            imageName = "notifications-email"
+        case .Timeline:
+            imageName = "notifications-bell"
+        case .Device:
+            imageName = "notifications-phone"
+        }
```
Comment (astralbodies) [inline WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingStreamsViewController.swift]: Maybe shortcut and return nil? Just thinking that passing in a nil string to UIImage(named:) might throw an error or exception.

Comment (astralbodies) [inline WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingStreamsViewController.swift]: On second thought this is an enum. There isn't a default case if all are accounted for. I'll let you decide. :smile:

Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingStreamsViewController.swift]: In Swift, if you don't add a case for every single (enum case), the compiler will kick your head off. No need to shortcut, in this land.

Thanks for the paranoia!! :dancer:

**Comment by astralbodies** [comment]
:shipit: 

Sorry forgot to squirrel it!

**Comment by jleandroperez** [comment]
Thank you Aaron!

**Pull request merged**